test/services/EnglishUIBuilderSpec.scala

```diff
@@ -961,12 +961,103 @@ class EnglishUIBuilderSpec extends BaseSpec with ProcessJson with EnglishLanguag
     val page = Page(Process.StartStanzaId, "/test-page", stanzas :+ KeyedStanza("5", input1), Seq.empty)
     val inputCurrencyPoundsOnly = models.ocelot.stanzas.CurrencyPoundsOnlyInput(inputNext, inputPhrase, Some(helpPhrase), label ="inputPounds", None, stack = false)
     val pagePoundsOnly = Page(Process.StartStanzaId, "/test-page", stanzas :+ KeyedStanza("5", inputCurrencyPoundsOnly), Seq.empty)
+    val inputText = models.ocelot.stanzas.TextInput(inputNext, inputPhrase, Some(helpPhrase), label ="inputText", None, stack = false)
+    val pageText = Page(Process.StartStanzaId, "/test-page", stanzas :+ KeyedStanza("5", inputText), Seq.empty)
+    val inputNumber = models.ocelot.stanzas.TextInput(inputNext, inputPhrase, Some(helpPhrase), label ="inputNumber", None, stack = false)
```

Comment:
Should we be using NumberInput rather than TextInput here?

Reply:
No TextInput allows a broader range of passphrases